server/routerlicious/packages/services/src/redis.ts

```diff
@@ -44,18 +44,30 @@ export class RedisCache implements ICache {
 	}
 
 	public async get(key: string): Promise<string> {
-		return this.client.get(this.getKey(key));
+		try {
+			return this.client.get(this.getKey(key));
+		} catch (error) {
+			// Cut off the key if it is too long
+			const keyShortened: string = key.length > 20 ? `${key.substring(0, 20)}...` : key;
+			Lumberjack.error(`Error getting ${keyShortened} from cache.`, undefined, error);
+			throw error;
+		}
 	}
 
 	public async set(key: string, value: string, expireAfterSeconds?: number): Promise<void> {
-		const result = await this.client.set(
-			this.getKey(key),
-			value,
-			"EX",
-			expireAfterSeconds ?? this.expireAfterSeconds,
-		);
-		if (result !== "OK") {
-			throw new Error(result);
+		try {
+			const result = await this.client.set(
+				this.getKey(key),
+				value,
+				"EX",
+				expireAfterSeconds ?? this.expireAfterSeconds,
+			);
+			if (result !== "OK") {
+				throw new Error(result);
+			}
+		} catch (error) {
+			Lumberjack.error(`Error setting ${key} in cache.`, undefined, error);
```

Comment:
Would `error` contains customer content?

Reply:
Hmm I'm not sure about this. Zach mentioned that it can contain JIT tokens, which isn't a security issue. I think it can also contain document Id's or some other Id's.